flatten-docs.py

```
@@ -0,0 +1,287 @@
+
+    Args:
+        html_file: Current location of the HTML file (absolute path)
+        version_dir: Version directory being processed (absolute path)
+        new_file_path: Future location relative to version_dir
+
+    Returns:
+        Number of changes made
+    """
+    try:
+        content = html_file.read_text(encoding='utf-8')
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)